### ISSUE TITLE
[MultiComboBox]: Accessibility: Duplicate aria-describedby property for multiple comboboxes

### ISSUE THREAD

**Issue by abhijeet181088** [SAP/ui5-webcomponents#9012]
### Describe the bug

If there are multiple MultiComboBoxes present on a page, the **"aria-describedby"** property of the "input" element within that is duplicate with value **"ui5-multi-combobox-hiddenText-nMore"**. This violates the accessibility guidelines.

Code snippet of input box inside MultiCombobox:

`<input id="ui5-multi-combobox-input" inner-input="" role="combobox" aria-haspopup="listbox" aria-autocomplete="both" data-sap-focus-ref="" placeholder="" value-state="None" aria-expanded="false" **aria-describedby="ui5-multi-combobox-hiddenText-nMore"** aria-required="false" aria-label="Line of Business:">`

### Isolated Example

https://stackblitz.com/edit/github-pngbdn?file=src%2FApp.tsx

### Reproduction steps

1. Create multiple MultiComboBoxes on a page. 
2. Inspect the "input" element within both MultiComboBoxes
3. They have same aria-describedby property as **"ui5-multi-combobox-hiddenText-nMore"**.
...

### Expected Behaviour

aria-describedby property should be unique for different components.

### Screenshots or Videos

<img width="1561" alt="AriaDescribedBy" src="https://github.com/SAP/ui5-webcomponents-react/assets/26282517/b8cd2964-5605-4181-ad6e-51de6b6118d2">

### UI5 Web Components for React Version

1.28.1

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

macOS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,
Could you please look into this issue? The problem reproduces following the described steps.

Regards,
Todor

**Comment by niyap** [comment]
Hello @abhijeet181088,

There is no violations of the accessibility guidelines as that HTML element is part of the shadow DOM of the component. There is no problem to have elements with similar id, when they are part of different shadow DOM.
Even the HTML input element which is part of the ui5-multi-combobox component has the same id for each instance of the component "ui5-multi-combobox-input"

Kind Regards,
Niya
